docs/introduction/media/media.md

```diff
@@ -0,0 +1,16 @@
+# Media
```

Comment:
Maybe make the sub-headers (e.g. `Blogs`, `Recorded Talks`, etc.) a different size than `Media`?

Reply:
sure thing